extensions/wikia/CookieSyncer/CookieSyncer.setup.php

```diff
@@ -0,0 +1,11 @@
+<?php
+$wgExtensionCredits['other'][] = [
+	'name' => 'CookieSyncer Extension',
+	'author' => [
+		'Platform Team @Wikia',
+	],
+	'descriptionmsg' => 'cookiesyncer-tag-desc',
```

Comment:
Isn't there supposed to be a i18n file with this message?

Reply:
Thankfully we don't need a message which carries creating a new Crowdin project. Instead, I used a plain `description`.